packages/genesis/README.md

```diff
@@ -15,6 +15,7 @@ This module provides access to Ethereum genesis state for the following networks
 - Goerli
```

Comment:
Could we drop Goerli?

Reply:
LTS for it expired ~1year ago: https://blog.ethereum.org/2023/11/30/goerli-lts-update